src/Plivo/Client/SystemHttpClient.cs

```diff
@@ -109,6 +110,13 @@ public SystemHttpClient(BasicAuth basicAuth, Dictionary<string, string> proxySer
             var baseServerUri = string.IsNullOrEmpty(baseUri) ? "https://api.plivo.com/" + Version.ApiVersion  : baseUri;
             _client.BaseAddress = new Uri(baseServerUri + "/");
 
+        
+            _callInsightsclient = new System.Net.Http.HttpClient(httpClientHandler);
+            _callInsightsclient.DefaultRequestHeaders.Authorization = authHeader;
+            _callInsightsclient.DefaultRequestHeaders.Add("User-Agent", "plivo-dotnet/" + ThisAssembly.AssemblyVersion);
+            var callInsightsBaseServerUri = "https://stats.plivo.com/" + Version.ApiVersion;
```

Comment:
**Version.ApiVersion** is being set and used in line 109 
`var baseServerUri = string.IsNullOrEmpty(baseUri) ? "https://api.plivo.com/" + Version.ApiVersion  : baseUri;`.

Say Version.ApiVersion is changed by other team, Our API calls will be failed as ours will be in /v1. So, better to have a variable of our own or append /v1 directly